### ISSUE TITLE
Test drive docs and issues templates

### ISSUE THREAD

**Issue by pirog** [thinktandem/tandem#14]
Before we start getting crazy with the new toys let's give them a shakedown cruise. Take a few hours each over the next week to do the following:

## 1. Go through the journey

Start at https://github.com/thinktandem/tandem and go through the flow presented in the README. In particular i mean step through and read all the documentation that is presented to you in that journey and add any notes, suggestions or feedback in the "Notes" section below.  I've added my own already so just follow the format there. 

## 2. Test drive the new issues

Head over to https://github.com/thinktandem/tandem/issues/new/choose and try filling out each of the issue types. Try to actually use a real task or proposal for something that you have in mind. Just filling them out so we can all compare results will be illuminating. Give particular care to the proposal issue since that is the most complex and important. Might even be good to start with it.

Beyond test driving the formats themselves drop some feedback in the "Notes" section below.

## Guidelines

While any feedback is good, here are some guidelines that might help you focus on the two tasks above:

0. Before you start writing your feedback check the notes section below. If someone else has already suggested something you were going to suggest just append your name to the bullet point and feel free to extend the comment with any additional things. 
1. Don't suggest any **HUGE** changes like _we should revamp the entire process_ try to keep the feedback narrowly scoped eg so they can be ultimately turned into an actionable task that shouldnt take more than a few hours
2. Definitely pay attention to clarity, if a section or procedure is not immediately clear, could be expressed better or simpler: _flag that_. If you think a section is lacking some key details, suggest what you think is lacking.
3. I've dropped a bunch of @todos that you will see at various places in your journey. These are areas to give extra attention.
4. There are no doubt countless grammar problems, typos, etc and if we flag all of those we will get overwhelmed. If you want to open a PR with fixes for those things i would do so. Take care here though because there very well could be an open PR with fixes that already exists. Check out the PRs first and if there is a grammar-ish PR already open just contribute directly to that so we can minimize collisions and duplicate work.
5. Remember, this is all a first draft of stuff so its very likely i did something wrong or poorly eg dont be afraid to shit on my parade.
6. If your feedback is not in by our Friday meeting on 6/28 then any suggestion you have will have to be considered later.

## Tips

* Remember, you've given yourself only a few hours so focus on the most important and obvious things. Don't worry about expressing an idea or suggestion at maximum depth, just surface enough so we can get the gist and its enough to refine the idea or suggestion over the coming week.
* Try to find a good time every week where you can dedicate a few hours in one sitting to Tandem work. This will help get you into a groove and help you start setting aside consistent "sacred" time. I personally would recommend east coast people do it pre-stand up and west coast people do it post-5pm EDT.

### README

* Could use more pop culture references - @pirog 
* Incorporate this video https://www.youtube.com/watch?v=UeAECmaJ7Ms and mention something about Ace of Basses - @labboy0276 
* *Getting Started:* I had to click through each link before figuring out I'm just being asked to read through the 2.0 docs. Summarize the reader should read through the 2.0 docs, and perhaps break the list into 2 parts, with the specific context above the 2.0 expectations. @mikemilano @reynoldsalec 

### Improving Tandem Docs

* I think the management of the process needs A LOT more detail. Particularly, it would be nice to have meeting agendas for each kind of meeting we do depending on where we are in the dev cycle - @pirog 
* I think it would also be great to clarify who runs the meetings and what the expectations around management are there eg backlog grooming, etc - @pirog 
* It might also be great, especially early on, to end every improvement cycle with a "mini cycle" similar to what we are doing now so we can always be improving this process. - @pirog 
* In regards to 
```
Every proposal and action item should be assigned to at least two people. This should help ensure momentum via redundancy and also makes sure that at least two people know about the thing 
```
I have a concern that we could run into an issue with a lack of clarity on who is primarly advancing an issue, how do we intend to handle that potential issue here? - @dustinleblanc 
* Our hardware / tools docs should probably be updated to reflect the greater Linux-ification of our dev team. It makes development with Lando oodles faster and hyperdrive supports debian-ish Linux distros pretty widely. the suggested versions of OSX and minimum memory are out of date too. - @dustinleblanc 
* Would like to develop more precision around prioritization of issues; right now most relevant section seems to [Managing the Process](https://docs.thinktandem.io/guides/improve-tandem.html#managing-the-process), which designates the Friday meeting as a time to designate a focus. We also have the "high sparrow" who presumably will be managing the board a bit, and the basic guidelines for issues help outline what makes a "good issue". - @reynoldsalec
* [Choosing and Working on Issues](https://docs.thinktandem.io/guides/improve-tandem.html#choosing-and-working-on-issues) may be too broad...how do we encourage people to work on things that motivate them AND move forward together as a unit? - @reynoldsalec 

### Proposal Issue Template

* I think more clarity around the "parts of the business" section would be good. Particularly it might be nice to have a more exhaustive list of parts, what they mean specifically and the form an answer should take. the @TODO around marketing/sales channels and audiences is a good example of what i mean. - @pirog 
* We still need the spreadsheet so we can prioritize our proposals - @pirog 
* Can we change the "number" sections on proposal templates to either header tags or bold sections? Makes it easier for me to browse an issue quickly and easier to use as a template (you can just delete the instruction/sample text and write your content below the header) - @reynoldsalec 
* I feel like items 2/3 are somewhat similar; for me "how we make it right" and describing a more automated solution are often answered by the same text. Should we consolidate? Felt like I was writing the same thing a couple times on other sections as well. - @reynoldsalec 
* Assuming all the 1-40 ratings supposed to be Fibonacci? - @reynoldsalec 

### Other issue Templates

* Just a question on the scripts section in Understand the Structure.  These scripts should be in the master templates (I am not sure if they are).  However are these just scripts people can grab and throw in their own repos?  Does that divert from our mission to have a handful of master templates that all projects will be spun off from?  - @labboy0276 

### Contributing Code

* In the section about testing and where you mention ```lando test``` should we link to one of the master templates where it shows what tests we run?  Or is that just for this repo? Right now it is vague to an outsider @labboy0276

**Comment by pirog** [comment]
@everyone, 

i've revamped our process docs and issue templates based around our conversation on Friday about simplifying things. I've also tried to incorporate some of the more action feedback here. I'd review your own feedback above and see if its been addressed. If not, and you think its something that should be surfaced if open up either a discussion or task so we can consider it!

beyond that im going to close out this and other issues so we have a clean slate to begin.
